.idea/codeStyles/Project.xml

```diff
@@ -3,520 +3,15 @@
     <option name="OTHER_INDENT_OPTIONS">
       <value>
         <option name="INDENT_SIZE" value="2" />
-        <option name="CONTINUATION_INDENT_SIZE" value="4" />
         <option name="TAB_SIZE" value="2" />
+        <option name="SMART_TABS" value="true" />
       </value>
     </option>
-    <option name="RIGHT_MARGIN" value="100" />
```

Comment:
These changes seem unrelated to the docs fix. Would you mind removing them from the PR (or perhaps open a new PR just for IDEA project changes)?